web/src/pages/services.tsx

```diff
@@ -4,6 +4,45 @@ import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 import { ServiceList } from '../components/services-page/service-list';
 
+const boxesData = {
+  dividedImage: 'ServicesSplit.png',
+  items: [
+    {
+      name: 'programming',
+      title: 'Applikasjons- og programvareutvikling',
+      description:
+        'Har du ideer for en applikasjon eller trenger kanskje bedriften din et program for en smidigere hverdag? Vi i Bjerk kan hjelpe deg med alt fra utvikling og integrering av programvare til applikasjoner slik at arbeidshverdagen din blir automatisert og dermed mer effektiv. Vår meget dyktige løsningsarkitekt og Head of Developing Team - Simen - kan hjelpe deg med å finne den beste løsningen for din ide.',
+      url: {
+        navigationLink: '',
+        linkText: 'Book et møte med Simen',
+      },
+      image: 'SimenBox.png',
+    },
+    {
+      name: 'strategy',
+      title: 'Strategisk rådgivning',
+      description:
+        'Våre strategiske rådgivere vil hjelpe deg med å fokusere på produktet bedriften din har. Vi diskuterer gjerne dine behov med tanke på kartlegging av både prisstrategi og lanseringsstrategi. Bjerk bistår deg også med å bygge et sterkt og konkurransedyktig team som vil bidra til at bedriften din når potensialet sitt. Snakk med Niklas om hvordan du og din bedrift kommer seg over den digitale dørstokken.',
+      url: {
+        navigationLink: '',
+        linkText: 'Book et møte med Niklas',
+      },
+      image: 'NiklasBox.png',
```

Comment:
We probably don't want `png`, and we probably don't want to combine the elements like shown (picture of person, illustration and text).

They should be three different elements.

Reply:
@cobraz Do I need to make a responsive design, for example, with setting parameters where the employee's photo will be located relative to the icon (in which of the four corners) and the text below (left / right)? Or just choose your own styles for each of the tiles in the template?